Formula/bear.rb

```diff
@@ -20,6 +20,7 @@ class Bear < Formula
   depends_on "grpc"
   depends_on macos: :catalina
   depends_on "nlohmann-json"
+  depends_on "protobuf"
```

Comment:
This isn't necessary; `bear` only depends on `protobuf` through `grpc`. `protobuf` is not listed as a dependency in the [upstream build docs](https://github.com/rizsotto/Bear/blob/master/INSTALL.md).

What you need to do is give `bear` a revision bump, not add this dependency.

Reply:
It does seem to try and access the files in the linkage check though.

Reply:
`protoc` in that bear build docs is this `protobuf`.

Reply:
You're looking at the wrong place; you need to be looking at the part that says [dependencies](https://github.com/rizsotto/Bear/blob/master/INSTALL.md#dependencies), which lists neither `protoc` nor `protobuf`.

I'd rather `protobuf` weren't listed, because if `bear` stops depending on `grpc`, we'd only need to remember to remove the `depends_on "grpc"` line rather than both `depends_on "grpc"` and `depends_on "protobuf"`. The `depends_on "protobuf"` adds nothing to the formula; it only makes maintenance of this formula slightly more complicated.

Also, regardless of whether you want to keep the `depends_on "protobuf"` or not, you still need to give this a revision bump.

Reply:
I bumped bear revision and remove `protobuf`.